docs/specs/meta-clients/web3modal/api.md

```diff
@@ -1,142 +1,248 @@
+All API endpoints expect the following simple (i.e. non-preflight-dependent) headers:
 
-```ts
-interface Headers {
-  "x-project-id": string;
-  "x-sdk-type": SdkType;
-  "x-sdk-version": SdkVersion;
-}
-```
+- `Accept-Encoding`
+  - Options: `gzip | br`
+  - Example: `Accept-Encoding: gzip, br`
 
-## Methods
+## API Endpoints
 
-### GET `/getWallets`
+### `POST /v2/wallets`
 
-Returns total count and wallet data, based on provided body inputs. See [types](./types.md).
+Fetches wallet data based on following request body provided as a stringified JSON object (`Content-Type: text/plain`).
 
-```ts
-interface Request {
-  headers: Headers;
-}
+#### Request Body
 
-interface UrlQueryParams {
-  page: string; // eg. 1
-  entries: string; // eg. 100
-  search?: string; // eg. MetaMa...
-  include?: string; // eg. id1,id2,id3
-  exclude?: string; // eg. id1,id2,id3
-  chains?: string; // eg. eip155:1,solana:5eykt4UsFv8P8NJdTREpY1vzqKqZKvdp
-  platform?: "ios" | "android";
+```typescript
+{
+  "page": number, // Required - The page number of results (e.g., `1`)
+  "entries": number, // Required - Number of entries per page (e.g., `100`)
+  "search": string | null, // Optional search term (e.g., `MetaMask`)
+  "include": string | null, // Optional comma-separated list of wallet IDs to include
+  "exclude": string | null, // Optional comma-separated list of wallet IDs to exclude
+  "chains": string | null, // Optional comma-separated list of chain identifiers (e.g., `eip155:1`)
+  "platform": string | null // Optional platform filter (`ios` or `android`)
 }
+```
+
+#### Response
 
-interface Response {
-  count: number;
-  data: Array<{
-    id: string;
-    name: string;
-    homepage: string;
-    image_id: string;
-    order: number;
-    mobile_link: string | null;
-    desktop_link: string | null;
-    webapp_link: string | null;
-    app_store: string | null;
-    play_store: string | null;
-    injected:
-      | {
-          namespace: string;
-          injected_id: string;
-        }[]
-      | null;
-  }>;
+```typescript
+{
+  "count": number,
+  "data": [
+    {
+      "id": string,
+      "name": string,
+      "homepage": string,
+      "image_id": string,
+      "order": number,
+      "desktop_link": string | null,
+      "webapp_link": string | null,
+      "app_store": string | null,
+      "play_store": string | null,
+      "mobile": {
+        deep_link: string | null,
+        universal_link: string | null
+      }
+      "injected": [
+        {
+          "namespace": string,
+          "injected_id": string
+        }
+      ] | null
+    }
+  ]
 }
 ```
 
-### GET `/getWalletImage/:wallet_id`
+### `GET /v2/ios-data`
 
-Returns image logo for requested `wallet_id`. See [types](./types.md).
+Returns a list of iOS-facing wallet data, including the public Explorer ID and iOS deep link schema.
 
-```ts
-interface Request {
-  headers: Headers;
-}
+#### Response
 
-type Response = Blob;
+```typescript
```

Comment:
How many entries is this going to return? Why can't the count be limited by the requester? Why isn't this part of `/getWallets`?